src/Model/EventParticipation.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
     /** {@inheritDoc} */
     public static function getAvailableStatuses()
     {
-        return parent::getAvailableStatuses() + [static::STATUS_NONE, static::STATUS_ORGANIZER];
+        return array_merge(parent::getAvailableStatuses(), [static::STATUS_NONE, static::STATUS_ORGANIZER]);
```

Comment:
strange, as `+` between two arrays is supposed to merge two arrays, but php I guess. :trollface: